package.json

```diff
@@ -4,7 +4,7 @@
  "displayName": "Upbound",
  "preview": true,
  "publisher": "upboundio",
- "version": "0.0.7",
+ "version": "0.0.1",
```

Comment:
Could we bump this to '0.0.2'? We still need to add in release branches (and automation around them) so this its manually at the moment.

Reply:
done